integration-tests/tests/src/performance-tests/property-reads.ts

```diff
@@ -16,53 +16,59 @@
 //
 ////////////////////////////////////////////////////////////////////////////
 
-import Realm, { ObjectSchema, PropertySchema } from "realm";
+import { expect } from "chai";
+import Realm, { BSON, ObjectSchema, PropertySchema, PropertySchemaShorthand } from "realm";
 
 import { describePerformance } from "../utils/benchmark";
 
 type Value = ((realm: Realm) => unknown) | unknown;
 
-function getTypeName(type: Realm.PropertySchemaShorthand | Realm.PropertySchema) {
-  if (typeof type === "object") {
-    const prefix = type.optional ? "optional " : "";
-    if (type.objectType) {
-      return prefix + `${type.type}<${type.objectType}>`;
-    } else {
-      return prefix + type.type;
-    }
-  } else {
-    return type;
+const COLLECTION_MARKERS: Readonly<Record<string, string>> = {
```

Comment:
Widening the type here for better ergonomics when bracket accessing it by an unknown string (and that's the only way it's used in this file).